src/common/logger/tests/logger_test.cpp

```diff
@@ -105,10 +75,10 @@ TEST_F(LoggerConstructorTest, MacOSLoggerConstructor)
 
     auto sinks = current_logger->sinks();
     ASSERT_FALSE(sinks.empty());
-    auto syslog_sink = std::dynamic_pointer_cast<spdlog::sinks::syslog_sink_mt>(sinks[0]);
-    EXPECT_NE(syslog_sink, nullptr);
+    auto systemd_sink = std::dynamic_pointer_cast<spdlog::sinks::stdout_color_sink_mt>(sinks[0]);
```

Comment:
And here, maybe we could just name the variable `sink`, since we're no longer using systemd or syslog. What do you think?

Reply:
Done